In [61]:
name = 'Anna'
hourly_wage = 10
hours = 40

In [62]:
float(hourly_wage)
int(hours)

40

In [63]:
weekly_pay = hourly_wage * hours
print(f"{name} earns {weekly_pay} per week")

Anna earns 400 per week


In [64]:
temps = [61, 65, 72, 58, 70, 74, 68]
high = 0
for s in temps:
    if s > 65:
        high = high + 1
print(high)

4


In [65]:
def to_celsius(f):
    return (f - 32) * 5/9

print(round(to_celsius(temps[0]), 1))

16.1


In [66]:
movie = {"title": "Dune", "year": 1965, "rating": 10}
for key, value in movie.items():
    print(key, ":", value)

title : Dune
year : 1965
rating : 10


In [67]:
import pandas as pd
df = pd.read_csv("chickwts1.csv")
df.head()

,weight,feed
0,179,horsebean
1,160,horsebean
2,136,horsebean
3,227,horsebean
4,217,horsebean


In [68]:
df.shape

(71, 2)

In [69]:
df.describe()

,weight
count,71.000000
mean,261.309859
std,78.073700
min,108.000000
25%,204.500000
50%,258.000000
75%,323.500000
max,423.000000


what does the describe() output tell you about chick weights (typical weight, spread, anything surprising)?

The typcal weight is relatively high as indicated by the mean and 50% and 75% percentile relatively high values. The spread is wide as indicated by the std deviation. It is suprising that across all feed samples that the weight is hgiher than the avereage value. Wigh a low weight of 108 in the sample, most of the population experienced significant weight gain.



https://github.com/chillyvee70/cs82a-portfolio/tree/main